cli/data_masking_config_file_parser.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import List, Any, Dict
+
+import yaml
+from pydantic import BaseModel
+
+
+class ColumnMaskingIdentity(BaseModel):
```

Comment:
In this case I would name it as `ColumnMaskingConfig`, because it is not config per single identity (identity may be `role` or `user`)